internal/cortex/frontend/transport/utils.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) The Cortex Authors.
```

Comment:
nit: let's make a note that the utils here are generated by us, otherwise it might be confusing for offsprings to think they are from OSS repo